examples/iam/variables.tf

```diff
@@ -0,0 +1,13 @@
+variable "project_id" {
+  type = string
+  description = "The Project ID where to create the Folder"
```

Comment:
Folders are not created inside projects.

Reply:
Is needed to activate service when not impersonating Service Account. Changed variable name to billing_project_id.